ballot_box/templates/_macros.html

```diff
@@ -11,7 +11,7 @@
 {%- endmacro %}
 
 {% macro ballot_panel_template(ballot, hr=True) -%}
-  <div class="panel panel-default">
+  <div class="panel panel-default{% if not g.user.can_candidate_signup(ballot) %} panel-inactive{% endif %}">
```

Comment:
Tohle makro je pouzito na mnoha mistech, takze to ma vedlesi ucinky. Spis bych to udelal formou parametru `inactive=False`, ktery by se predaval jako `True` jen z `candidate_signup.html`.